config/langs.toml

```diff
@@ -691,6 +691,30 @@ fn main() {
 }
 '''
 
+[Sed]
+size    = '222 KiB'
+version = '4.8'
+website = 'https://www.gnu.org/software/sed/manual/sed.html'
```

Comment:
We generally link to homepages, not manuals, so `https://www.gnu.org/software/sed/`